### PR TITLE
Add new slider + Add CSS Modules

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request?
As title says.

#### What problem is this solving?
The old slider wasn't working properly.

### Extra information
The autoplay functionality depends in the acception of [this PR](https://github.com/vtex-apps/slider/pull/4)

#### How should this be manually tested?
[Workspace](https://newcarousel--storecomponents.myvtex.com/)

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Carousel.tsx [hunk 1]
```diff
@@ -198,43 +213,95 @@ export default class Carousel extends Component<Props> {
...
+    }
+  }
+
+  public handleChangeSlide = (i: number): void => {
+    this.setState({ currentSlide: i })
+  }
+
+  public ArrowRender: React.StatelessComponent<ArrowProps> = ({ orientation, onClick }: ArrowProps) => {
+    return (
+      <div className="vtex-carousel__arrow pointer z-1" onClick={onClick}>
```
Comment (theomoura) [inline react/Carousel.tsx]: `vtex-carousel__arrow`? couldn't use css modules?

File: react/Banner.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import { Link, withRuntimeContext } from 'vtex.render-runtime'
```
Comment (JuanAlmeida12) [inline react/Banner.tsx]: You can change this to use hook `useRuntime`

File: react/Carousel.tsx [hunk 3]
```diff
@@ -198,43 +213,95 @@ export default class Carousel extends Component<Props> {
     }
   }
 
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
```
Comment (jgfidelis) [inline react/Carousel.tsx]: no need for constructor

File: react/Carousel.tsx [hunk 4]
```diff
@@ -198,43 +213,95 @@ export default class Carousel extends Component<Props> {
...
+
+  public ArrowRender: React.StatelessComponent<ArrowProps> = ({ orientation, onClick }: ArrowProps) => {
+    return (
+      <div className="vtex-carousel__arrow pointer z-1" onClick={onClick}>
+        <IconCaret orientation={orientation} size={32} />
+      </div>
+    )
+  }
+
+  public ArrowContainerRender: React.StatelessComponent<ArrowContainerProps> = ({ children }: ArrowContainerProps) => {
```
Comment (jgfidelis) [inline react/Carousel.tsx]: `StatelessComponent` but your component has a state???

**Pull request merged**